cpplite/cpplite.debugger/src/org/netbeans/modules/cpplite/debugger/CPPLiteDebuggerConfig.java

```diff
@@ -50,6 +53,14 @@ public File getDirectory() {
         return directory;
     }
 
+    public boolean isAttach() {
+        return processId.isPresent();
+    }
+
+    public long getProcessId() {
+        return processId.get();
```

Comment:
What happens when the `Optional` isn't set? A `NullPointerException`?

Reply:
There is `isAttach()` to test it. If it's false and you still ask for the process ID, it fails.
I can replace it with `Long getProcessId()`, which may return `null`. It'd be more straightforward.

Reply:
It's documented and checked explicitly.